src/sage/env.py

```diff
@@ -151,13 +151,11 @@ def var(key: str, *fallbacks: Optional[str], force: bool = False) -> Optional[st
 SAGE_VERSION_BANNER = var("SAGE_VERSION_BANNER", version.banner)
 
 # virtual environment where sagelib is installed
-SAGE_VENV = var("SAGE_VENV", os.path.abspath(sys.prefix))
 SAGE_LIB = var("SAGE_LIB", os.path.dirname(os.path.dirname(__file__)))
 SAGE_EXTCODE = var("SAGE_EXTCODE", join(SAGE_LIB, "sage", "ext_data"))
-SAGE_VENV_SPKG_INST = var("SAGE_VENV_SPKG_INST", join(SAGE_VENV, "var", "lib", "sage", "installed"))
 
 # prefix hierarchy where non-Python packages are installed
-SAGE_LOCAL = var("SAGE_LOCAL", SAGE_VENV)
+SAGE_LOCAL = var("SAGE_LOCAL")
```

Comment:
any reason to drop the `sys.prefix` fallback?

Reply:
`SAGE_LOCAL` is set unconditionally in `config.py.in` (which gets used in `var`) to the prefix that was specified to meson. So there is no need for a fallback.